### DISCUSSION TITLE
Clarify discussions vs. issues

### DISCUSSION THREAD

**Discussion by tordans** [bryceco/GoMap#752] [General]
I wrote about it for iD and I think this is true for GoMap as well: https://github.com/openstreetmap/iD/discussions/8841#discussioncomment-2280089

One step would be to add an issue template like https://github.com/openstreetmap/iD/blob/develop/.github/ISSUE_TEMPLATE/config.yml

**Comment by tordans** [comment]
Also, one thing that might not be knows: I can subscribe to issues ("get a notification on every issue and any comment") but not to discussions. So everything written here, will be invisible to me unless I actively search for it; which I usually never do.

**Comment by bryceco** [comment]
This makes me think that I should simply disable discussions. I'm totally okay with discussions happening in issues, which avoid questions about where to post, and if they are more discoverable then that's a plus.

**Reply by tordans** [reply]
> This makes me think that I should simply disable discussions. 

That's what I would do for most of my projects, yes. They make sense for big projects that need a QA forum and have enough community activity to answer questions there.
